### DISCUSSION TITLE
posts/Go/golang-menu/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#321] [Announcements]
https://www.liwenzhou.com/posts/Go/golang-menu/ 

 李文周的Blog提供Go语言教程，golang中文教程,李文周博客,Go语言学习之路是我整理的一个免费教程和视频教程，七米外号娜扎、qimi、小王子、沙河娜扎，自己写的Go语言入门教程，非常适合新手入门学习go语言。是李文周的学习笔记。

**Comment by knifesoft** [comment]
七米，您好！请问这篇文章是怎么置顶🔝的？其中设置了什么参数？多谢。

**Comment by 850552586** [comment]
非常感谢七米老师的分享~

**Comment by 15372077356** [comment]
老刘牛逼

**Comment by cold-runner** [comment]
> 七米，您好！请问这篇文章是怎么置顶🔝的？其中设置了什么参数？多谢。

应该是设置Frontmatter中的weight属性，这个值越大，在页面显示的顺序越靠后

**Comment by HelloEyre** [comment]
> @15372077356
> 老刘牛逼

七米牛逼，讲的挺好的，语速和知识点都很清晰！

**Comment by HelloEyre** [comment]
> @15372077356
> 老刘牛逼
忘记说了，我虹桥*王鸡哥表示不服！

**Comment by hanzhiyan588** [comment]
Cookie和Session页面报错了

**Comment by For-The-Ideal** [comment]
测试
